### PR TITLE
Add comms on migration to single-region buckets

### LINKED ISSUE #34: Communicate transition to single-region bucket
From Ag3.12 and Af1.5 onwards.

Emphasise impact on direct downloads, as currently the API uses the single-region bucket directly.

### PR DESCRIPTION
**Pull request by ahernank**
* Resolves #34

### CODE REVIEW

File: _posts/2024-11-19-single-region-transition.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Update on Data Storage"
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:3]:
```suggestion
title:  "Migration to single-region GCS buckets"
```

File: _posts/2024-11-19-single-region-transition.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{% highlight python %}
+import malariagen_data
+
+malariagen_data.__version__
+
+{% endhighlight %}
+
+  - **If you use Google Colab:**
+      - Confirm you’re running the latest version of the API (v13.0.0 or newer).
+      - If you encounter an error due to being allocated a Virtual Machine (VM) outside of the US, go to `"Runtime" > "Disconnect and delete runtime"` to allocate a new VM within the US region.
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:26]:
```suggestion
```

Is this strictly necessary here?

File: _posts/2024-11-19-single-region-transition.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+2. **If you access data via data directly from GCS buckets:**
+- The following bucket name changes have been implemented:
+    - Ag data previously stored in `gs://vo_agam_release` is now in `gs://vo_agam_release_master_us_central1`
+    - Af data previously stored in `gs://vo_afun_release` is now in  `gs://vo_afun_release_master_us_central1`
+
+- All data has been mirrored to the new buckets. There is no need to download any data.
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:33]:
```suggestion
- All data has been mirrored to the new buckets. Please update any scripts or code to use the new bucket names.
```

File: _posts/2024-11-19-single-region-transition.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- The multi-region buckets (`vo_agam_release`, `vo_agam_release`) will be deprecated by November 30th, 2024. You might need to update any scripts accessing these buckets to use the new names.
+- All new data will only be available in the single-region buckets moving forward.
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:35]:
```suggestion
**N.B., the multi-region buckets (`vo_agam_release`, `vo_agam_release`) will be deprecated on November 30th, 2024. Please follow the instructions above to migrate to using single-region buckets before this date.**
```

Maybe make this a bit bolder so it's harder to miss?

File: _posts/2024-11-19-single-region-transition.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Update on Data Storage"
+tags: data
+---
+
+We are making schanges to how data is stored in the Vector Observatory (VObs).
+Currently, data is stored in multi-region Google Cloud Storage (GCS) buckets. After evaluating accessibility needs, we’ve decided to move the data to single-region buckets located in the United States (US). This transition reduces our running costs without affecting data availability or performance.
```
[the remark this answers is not in the document]
Comment (alimanfoo) [reply _posts/2024-11-19-single-region-transition.md:8]: Maybe be more specific about the region. Also not obvious what "after evaluating accessibility needs" means, maybe can leave out?

File: _posts/2024-11-19-single-region-transition.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+Due to this, we are starting to roll-out some changes:
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:10]:
```suggestion
To access data in the new single-region buckets, please follow the instructions below:
```

File: _posts/2024-11-19-single-region-transition.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+1. **If you access data via the malariagen_data API:**
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:12]:
```suggestion
1. **If you access data via the malariagen_data Python API:**
```

File: _posts/2024-11-19-single-region-transition.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  - Please ensure that you are only using API versions v13.0.0 or newer. If you install `malariagen_data` without specifying a version (e.g., using `%pip install malariagen_data`), the latest version will automatically be installed.
```
Comment (alimanfoo) [inline _posts/2024-11-19-single-region-transition.md:13]:
```suggestion
  - Please ensure that you are only using API versions v13.0.0 or newer. If you install `malariagen_data` without specifying a version (e.g., using `pip install malariagen_data`), the latest version will automatically be installed. If you have previously installed `malariagen_data`, please run `pip install --upgrade malariagen_data` to upgrade to the latest software version.
```

**Comment by ahernank** [comment]
Merging soon, unless any concerns.

**Pull request merged**